cypress/e2e/scenario-8-instance-composer.cy.js

```diff
@@ -1,778 +1,827 @@
-      cy.get('[data-action="link"]')
-        .trigger("mouseover")
-        .trigger("mousedown")
-        .trigger("mousemove", {
-          clientX: 900,
-          clientY: 300,
-        })
-        .trigger("mouseup");
-      cy.get("button").contains("Deploy").click();
-
-      // Check if only one row has been added to the table.
-      cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 6);
-      // await until all instances are being deployed and up
-      cy.get('[data-label="State"]', { timeout: 90000 })
-        .eq(0, { timeout: 90000 })
-        .should("have.text", "up");
-      cy.get(".pf-v5-c-nav__item").contains("Service Catalog").click();
-
-      // check if attribute was edited correctly
-      cy.get("#child-with-many-parents-service", { timeout: 60000 })
-        .contains("Show inventory")
-        .click();
-      cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 1);
-      cy.get('[data-label="State"]', { timeout: 90000 }).should(
-        "have.text",
-        "up",
-      );
-      cy.get("#expand-toggle0").click();
-      cy.get("button").contains("Attributes").click();
-      cy.get(
-        '[aria-label="Row-parent_entity"] > [data-label="active"] > .pf-v5-l-flex > div > .pf-v5-c-button',
-      )
-        .eq(0)
-        .should("have.text", "parent-service3");
-      cy.get(
-        '[aria-label="Row-parent_entity"] > [data-label="active"] > .pf-v5-l-flex > div > .pf-v5-c-button',
-      )
-        .eq(1)
-        .should("have.text", "new-parent-service3");
-    });
-
-    it("8.5 delete instance", () => {
-      // Select 'test' environment
-      cy.visit("/console/");
-      cy.get('[aria-label="Environment card"]')
-        .contains("lsm-frontend")
-        .click();
-      cy.get(".pf-v5-c-nav__item").contains("Service Catalog").click();
-
-      // click on Show Inventory on embedded-entity-service-extra, expect one instance already
-      cy.get("#parent-service", { timeout: 60000 })
-        .contains("Show inventory")
-        .click();
-      cy.get('[aria-label="ServiceInventory-Success"]').should("to.be.visible");
-      cy.get('[aria-label="InstanceRow-Intro"]').should("have.length", 6);
-
-      // click on kebab menu on embedded-entity-service-extra
-      cy.get('[aria-label="row actions toggle"]').eq(5).click();
-      cy.get("button").contains("Edit in Composer").click();
-      cy.get('[joint-selector="headerLabel"]')
-        .contains("parent-service")
-        .click();
-      cy.get('[data-action="delete"]').click();
-      cy.get("button").contains("Deploy").click();
-
-      // Check if only one row has been deleted to the table.
-      cy.get('[aria-label="InstanceRow-Intro"]', { timeout: 90000 }).should(
-        "have.length",
-        5,
-      );
-    });
-  });
-}
+//TODO: tests are commented out as there is ongoing redesign of the instance composer which will affect how the user interact with it and how e2e test should look like
```

Comment:
Just make certain they are only commented out on ISO8

Reply:
iso7 does have this scenario intact